test/exceptions/src/assert_fail.d

```diff
@@ -64,6 +64,7 @@ void testFloatingPoint()()
     test(1.5, 2.5, "1.5 != 2.5");
     test(float.max, -float.max, "3.40282e+38 != -3.40282e+38");
     test(double.max, -double.max, "1.79769e+308 != -1.79769e+308");
+    test(real.max, -real.max, "1.18973e+4932 != -1.18973e+4932");
```

Comment:
Please use another constant, `real` isn't 80-bit for most targets.

Reply:
Maybe this way ?

```
import std.conv : to;
//test(real.max, -real.max, "1.18973e+4932 != -1.18973e+4932");
test(real.max, -real.max, real.max.to!string ~ " != -" ~ real.max.to!string);
```

How shall I test this with a number which change depending of the platform ?

Reply:
Well, `real.max` seems totally arbitrary, so any nicely representable values should do (`5.25L` or whatever).

Reply:
Changed to:
```
test(5.25L, -5.25L "5.25 != -5.25");
```

Reply:
> Changed to:
> 
> ```
> test(5.25L, -5.25L, "5.25 != -5.25");
> ```

Somehow it returns this:
```
'-nan != -nan' != '5.25 != -5.25'
```
Any idea @kinke @Geod24 ?